Mobile/ShopLens/Droid/Activities/MainActivity.cs

```diff
@@ -102,69 +91,40 @@ protected override void OnCreate(Bundle savedInstanceState)
                 Resource.String.closeDrawer
             );
             drawerLayout.AddDrawerListener(drawerToggle);
-            SupportActionBar.SetHomeButtonEnabled(true);
+            SetSupportActionBar(toolbar);
```

Comment:
Maybe this was the reason why the drawer didn't show after hamburger button click?

Reply:
The reason was I just forgot to add 
`public override bool OnOptionsItemSelected(IMenuItem item)
        {
            drawerToggle.OnOptionsItemSelected(item);
            return base.OnOptionsItemSelected(item);
        }`